database-migration-tool/README.md

```
@@ -0,0 +1,311 @@
+db-migrate status
+
+# Rollback to version
+db-migrate down --to 20251212_add_indexes
+
+# Verify
+db-migrate status
+```
+
+## Schema Versioning
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)